index.html

```diff
@@ -0,0 +1,139 @@
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body class="index-body">
+    <div class="optionsheadercontainer">
+    <ul class="optionsheader">
+      <li><i><img class="icon1" src="./imgs/facebook-icon-png-774.png" alt="" href="#"></i></li>
+      <li><i><img class="icon2" src="./imgs/Twitter.png" alt="" href="#"></i></li>
+      <li class="optiontext1"><a href="#">English</a></li>
+      <li class="optiontext2"><a href="#">log in</a></li>
+    </ul>
+  </div>
+    <header class="header">
+        <div class="headerelement headerelement1"></div>
+        <nav class="headerelement headerelement2">
+          <div class="lines">
+            <div class="menuline line-1"></div>
+            <div class="menuline line-2"></div>
+            <div class="menuline line-3"></div>
+          </div>
+          <ul class="menuitems">
+            <li class="menuitem menuitem-1">
+                <a href="./index.html">Home</a>
+            </li>
+            <li class="menuitem menuitem-2">
+              <a href="./about.html">About</a>
+            </li>
+            <li class="menuitem menuitem-3">
+              <a href="#">lorem</a>
+            </li>
+            <li class="menuitem menuitem-4">
+              <a href="#">Ipsum</a>
+            </li>
+          </ul>
+          <div class="navmenubottom"></div>
+        </nav>
+        <nav class="header-desktop">
+          <div class="headerimgcontainer"><a href="./index.html"><img class="headerimg" src="./imgs/E3logowhite.png" alt=""></a></div>
+          <ul class="headercontainer">
+            <li class="header-desktop1"><a href="./index.html">Home</a></li>
+            <li class="header-desktop2"><a href="./about.html">About</a></li>
+            <li class="header-desktop3"><a href="#">Lorem</a></li>
+            <li class="header-desktop4"><a href="#">Ipsum</a></li>
+          </ul>
+          <div class="hbuttoncontainer"><button class="headerbutton">News</button></div>
+        </nav>
+      </header>
+      <main>
+        <div class="headline">
+          <p class="ph1">"Stay in the game!"</p>
+          <h1 class="p1">
+              Worldwide broadcast E3 2021
+          </h1>
+          <h2  class="pd1">
+            Worldwide broadcast E3 2021
+          </h2>
+          <p class="pheadline">
+            Hardware manufacturers, software developers, 
+            and publishers from the video game industry would have 
+            presented new and upcoming products. 
+          </p>
+          <h2 class="date">June.9.2020 (TUE) ~ 11 (FRY)</h2>
+          <p class="location">@ Los Angeles Convention Center, Los Angeles California</p>
+        </div>
+        <section class="events">
+          <h2 class="mainevents">Main Events</h2>
+          <h2 class="eventsline1">____</h2>
+          <div class="eventcard eventcard1">
+            <img class="eventimg" src="./imgs/SONY-SQUARE1.png" alt="">
+            <h2 class="event-tittle">Sony conference</h2>
```

Comment:
- [ ] According to `hx` hierarchy, it would be great if, after use `h2` in a father element, these children could use `h3`. I thinks this could improve the SEO for your page. Please change it. 🦊

Reply:
You're totally right, those h2 are h3 already. 😄